modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -56,17 +71,21 @@ HashTSDFVolumeCPU::HashTSDFVolumeCPU(float _voxelSize, cv::Matx44f _pose, float
     : HashTSDFVolume(_voxelSize, _pose, _raycastStepFactor, _truncDist, _maxWeight,
                      _truncateThreshold, _volumeUnitRes, _zFirstMemOrder)
 {
+    lastVolIndex = 0;
+    volUnitsData = cv::Mat(VOLUMES_SIZE, volDims.x * volDims.y * volDims.z, rawType<TsdfVoxel>());
```

Comment:
Why not call `reset` here instead?

Reply:
If you mean release() function, it will remove the memory of the Mat variable. But we need to have a small buffer instead of a cleaned buffer

Reply:
I mean `HashTSDFVolumeCPU::reset()` functions which consists of exactly the same 2 lines of code.